resident/resident-service/src/main/java/io/mosip/resident/config/Config.java

```diff
@@ -3,6 +3,9 @@
 import io.mosip.kernel.core.templatemanager.spi.TemplateManager;
 import io.mosip.kernel.keygenerator.bouncycastle.KeyGenerator;
 import io.mosip.kernel.templatemanager.velocity.impl.TemplateManagerImpl;
```

Comment:
Please inclue the Jira id in the exact format in the PR title, so PR reflect in the Jira id